master/internal/core.go

```diff
@@ -215,15 +215,15 @@ func (m *Master) getMasterLogs(c echo.Context) (interface{}, error) {
 // @ID get-raw-resource-allocation-csv
 // @Accept  json
 // @Produce  text/csv
-//nolint:lll
 // @Param   timestamp_after query string true "Start time to get allocations for (YYYY-MM-DDTHH:MM:SSZ format)"
-//nolint:lll
 // @Param   timestamp_before query string true "End time to get allocations for (YYYY-MM-DDTHH:MM:SSZ format)"
-//nolint:lll
 // @Success 200 {} string "A CSV file containing the fields experiment_id,kind,username,labels,slots,start_time,end_time,seconds"
-//nolint:godot
 // @Router /allocation/raw [get]
 // @Deprecated
+//
+//nolint:lll
+//nolint:lll
+//nolint:lll
```

Comment:
Wow, the linter was really hard on you! Sorry you had to deal with all this; it's not normally like this. I wonder why all these linting issues haven't come up before. Since you didn't modify many of these files other than the linting issues, it can't be that…